ansible/roles/os_config/tasks/memory_optimization.yml

```diff
@@ -3,28 +3,28 @@
     domain: root
     limit_type: soft
     limit_item: nofile
-    value: 200000
+    value: 1000000
```

Comment:
please describe when we discovered the problem .. what was the problem ..

Reply:
not aligned to our gravity installation and k8s installs settings.